src/components/app/details/triggerView/workflow/nodes/staticNode.tsx

```diff
@@ -19,19 +20,26 @@ export interface StaticNodeProps {
     regex?: string
     isRegex?: boolean
     primaryBranchAfterRegex?: string
+    handleGoToWorkFlowEditor?: (...args) => void
```

Comment:
Change this to `(...args) => void` to `(e?: any) => void`

Reply:
done